clusterloader2/pkg/measurement/common/scheduling_throughput.go

```diff
@@ -70,9 +70,12 @@ func (s *schedulingThroughputMeasurement) Execute(config *measurement.Measuremen
 		if err := selector.Parse(config.Params); err != nil {
 			return nil, err
 		}
-
+		measurmentInterval, err := util.GetDurationOrDefault(config.Params, "measurmentInterval", defaultWaitForPodsInterval)
```

Comment:
Would you mind creating a new const to replace `defaultWaitForPodsInterval`? 
The `defaultWaitForPodsInterval` is defined in other file, with main purpose of being the interval for "waitForPods" measurement. We shouldn't have a dependency like that - imagine someone changing wait for pods interval and having a side effect of changing how scheduler_throughput works, not really a best code :)

Reply:
I'm aware of relationship between `defaultWaitForPodsInterval` and `waitForPods`, however I'm not sure that we really want to remove this side-effect.

Aligning waitForPods and schedulingThroughput has a benefit that we have a shared precision, and that looks reasonable. If you feel bad about it, then I guess it should be refactored to some shared common default precision variable. WDYT?

Reply:
I don't agree that shared precision looks reasonable. It's not even a precision in case of the waitForPods. 
In the waitForPods this parameter only controls how often we check whether a certain condition has occurred. One could easily argue, that we could increase this parameter to 30s to reduce CPU usage of clusterloader2 and it would have no-impact on our tests.
On the other hand, increasing this param to 30s would completely break the scheduler throughput for smaller tests.

Reply:
Added independent default equal to 5s to be backward compatible.